src/olympia/addons/tests/test_admin.py

```diff
@@ -226,7 +226,7 @@ def test_can_list_with_addons_edit_permission(self):
         self.client.login(email=user.email)
         response = self.client.get(self.list_url, follow=True)
         assert response.status_code == 200
-        assert '/addon/bar-replacement/' in response.content
+        assert '/addon/bar-replacement/' in response.content.decode('utf-8')
```

Comment:
I've been wondering if we should adopt a best practise?  
`assert '/addon/bar-replacement/' in response.content` to
`assert b'/addon/bar-replacement/' in response.content` or
`assert '/addon/bar-replacement/' in response.content.decode('utf-8')` or
`assert '/addon/bar-replacement/' in force_text(response.content)`

Reply:
Been wondering about that too. I think any of the last 2 are best, but I sometimes just used the second one out of lazyness.

Reply:
yeah I've been inconsistently making different changes in different places too.  I guess it's fine.  (until it's not)